src/js/components/award/SummaryBar.jsx

```diff
@@ -4,21 +4,52 @@
  **/
 
 import React from 'react';
+import moment from 'moment';
 import * as Icons from '../sharedComponents/icons/Icons';
+import InfoSnippet from './InfoSnippet';
 
 const propTypes = {
-    selectedAward: React.PropTypes.object,
-    awardStatus: React.PropTypes.string
+    selectedAward: React.PropTypes.object
 };
 
 export default class SummaryBar extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.getStatus = this.getStatus.bind(this);
+    }
+
+    componentWillMount() {
+        this.getStatus(this.props.selectedAward);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.selectedAward !== this.props.selectedAward) {
+            this.getStatus(nextProps.selectedAward);
+        }
+    }
+
+    getStatus(award) {
+        const awardStart = moment(award.period_of_performance_start_date, 'MM-DD-YYYY');
+        const awardEnd = moment(award.period_of_performance_current_end_date, 'MM-DD-YYYY');
+        const current = moment();
+        let progress = "";
+        if (current.isBefore(awardStart)) {
+            progress = "Awarded";
+        }
+        else if (current.isAfter(awardEnd)) {
+            progress = "Complete";
+        }
+        else {
+            progress = "In Progress";
+        }
+        this.progress = progress;
```

Comment:
Why is the progress string stored as a class variable? This would make more sense in the component state.